.github/workflows/ci-v3.yml

```diff
@@ -35,8 +35,6 @@ jobs:
   publish-module:
     name: 'Publish Module to NPM'
     needs: test
-    # publish only when merged in master on original repo, not on PR
-    if: github.repository == 'TanStack/query' && github.ref == 'refs/heads/v3'
```

Comment:
This is a redundant check as it will always be evaluated to `true` (and therefore can be removed).